app/uk/gov/hmrc/trackingconsentfrontend/assets/src/config/config.ts

```diff
@@ -0,0 +1,21 @@
+export const getWelshLanguageEnabled = () => false
+
+const getPort = (): string | undefined => {
+    const scriptTag: HTMLScriptElement | null = document.querySelector('script[data-id="tracking-consent-frontend"]')
```

Comment:
I'm nervous about looking up the script tag in this way, it:

a) changes the interface with the services
b) allows any script tag to be wrongly identified as the tracking consent script tag

I think we should discuss the pros/cons of using the approach from my PR https://github.com/hmrc/tracking-consent-frontend/pull/82/files#diff-c50745f995c8fa68ee8fb65caa8781a4c7f5d5825a720d16dc76c357da88ab1aR12

I had previously discounted using the `currentScript` option, but actually it would keep more modern browsers safer so maybe we should use that and only fall back if it's not available.

Reply:
currentScript is not available in IE.

Reply:
In the meeting on 19 October 2020, we agreed the principle of using an identifier to look up the current script. We also talked about using getElementById. However, on reflection that may be a little more risky than above due to the fact that this will pollute the id namespace and potentially cause problems if any other elements are defined with that id, unless we use very unique id like a GUID. The code above will only cause problems if another script tag has been defined with data-id="tracking-consent-frontend", which seems unlikely in my opinion.